_config.yml

```diff
@@ -1,8 +1,8 @@
 # Site settings
 title: Blog about testing & documentation for KnapsackPro.com
-description: > # this means to ignore newlines until "baseurl:"
```

Comment:
the comment wasn't very clear, but I noticed that the newlines are ignore regardless of the `>` sign, so instead of updating it, I just removed the whole thing